# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/moz.build
# Commit: a48aff5f2555
# Full Hash: a48aff5f2555af61a9bc1ee0d77f6058f47d0501
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7e54a223001d -r a48aff5f2555 js/loader/moz.build
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/loader/moz.build	Thu Feb 17 15:57:52 2022 +0000
@@ -0,0 +1,26 @@
+# -*- Mode: python; indent-tabs-mode: nil; tab-width: 40 -*-
+# vim: set filetype=python:
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+EXPORTS.mozilla.dom += [
+    "LoadedScript.h",
+    "ModuleLoaderBase.h",
+    "ModuleLoadRequest.h",
+    "ModuleMapKey.h",
+    "ScriptKind.h",
+    "ScriptLoadRequest.h",
+]
+
+UNIFIED_SOURCES += [
+    "LoadedScript.cpp",
+    "ModuleLoaderBase.cpp",
+    "ModuleLoadRequest.cpp",
+    "ModuleMapKey.cpp",
+    "ScriptLoadRequest.cpp",
+]
+
+FINAL_LIBRARY = "xul"
+
+include("/ipc/chromium/chromium-config.mozbuild")